go.mod

```diff
@@ -24,17 +24,74 @@ require (
 	github.com/dell/gocsi v1.5.0
 	github.com/dell/gofsutil v1.7.0
 	github.com/dell/goscaleio v1.6.0
-	github.com/fsnotify/fsnotify v1.4.9
+	github.com/fsnotify/fsnotify v1.5.1
 	github.com/golang/protobuf v1.5.2
 	github.com/gorilla/mux v1.8.0
 	github.com/kubernetes-csi/csi-lib-utils v0.9.1
 	github.com/sirupsen/logrus v1.8.1
-	github.com/spf13/viper v1.8.1
+	github.com/spf13/viper v1.10.1
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/net v0.0.0-20211015210444-4f30a5c0130f
-	google.golang.org/grpc v1.42.0
+	google.golang.org/grpc v1.43.0
 	k8s.io/api v0.22.2
 	k8s.io/apimachinery v0.22.2
 	k8s.io/client-go v0.22.2
 	sigs.k8s.io/yaml v1.2.0
 )
+
+require (
```

Comment:
Just out of curiosity, what necessitates the massive replace block? I thought we just got rid of it?

Reply:
This is a require block, not a replace block. 
It's applied automatically in Go 1.17, because indirect dependencies are managed in their own "require" block. 
From the [release notes](https://go.dev/doc/go1.17) : "Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies."